### PR TITLE
fix rendering empty accordion, if no footer links are provided

### LINKED ISSUE #1836: axa-footer: renders empty accordion on mobile
Hi,

MyAXA portal uses` "@axa-ch/footer": "3.0.8"` and I noticed bug in mobile design. 

We are using only one "column" section, not two as yours demo.
Empty accordion is rendered on mobile resolution.

I tried to reproduce it with the latest version, but it is not possible because of another issue #1832 .

Repo to reproduce (branch `myaxa-version`):
https://github.com/branislav-remen/patterns-lib-footer/tree/myaxa-version

image
image

### PR DESCRIPTION
**Pull request by raphaellueckl**
Fixes #1836

This PR should probably not be merged, before the component versioning is fixed.

### CODE REVIEW

**Comment by danyball** [comment]
Azure failing?!

File: src/components/30-organisms/footer/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 4.1.2
+
+- Do not render an empty accordion on mobile if no content is given. #1836
```
Comment (01231) [inline src/components/30-organisms/footer/CHANGELOG.md]: Would write: Only render accordion on mobile when content is given

File: src/components/30-organisms/footer/demo.js [hunk 2]
```diff
@@ -62,5 +62,10 @@ storiesOf('Components|Footer/Demos', module)
       span.innerText = `Last clicked link: ${ev.detail}`;
     });
 
+    return wrapper;
+  })
+  .add('Without content', () => {
+    const wrapper = document.createElement('div');
+    wrapper.innerHTML = `<axa-text>This is an emtpy footer. Just for testing.</axa-text><axa-footer></axa-footer>`;
```
Comment (01231) [inline src/components/30-organisms/footer/demo.js]: not very eloquent in my opinion. I would add a footer like this but with the header `AXA WELTWEIT` in addition (only the header):
Screenshot 2020-07-23 at 09 46 14

And then write something like: No carousel should be rendered on mobile for `AXA WELTWEIT`

**Review by 01231 — CHANGES_REQUESTED** [review]

File: src/components/30-organisms/footer/demo.js (hunk 2 above)
Comment (danyball) [reply src/components/30-organisms/footer/demo.js]: This is not a common use case. Who wants to have a column with a header but without content? See the issue for that PR. The user do not use column 2. No header, no items.

File: src/components/30-organisms/footer/CHANGELOG.md (hunk 1 above)
Comment (danyball) [reply src/components/30-organisms/footer/CHANGELOG.md]: Wont fix. Minor.

Comment (01231) [reply src/components/30-organisms/footer/CHANGELOG.md]: Why not? `ctrl C` + `ctrl V`
Double negative is difficult to read.

File: src/components/30-organisms/footer/demo.js (hunk 2 above)
Comment (01231) [reply src/components/30-organisms/footer/demo.js]: Then don't add the header. I just don't like that there is just a blue square in the demo. You can't tell what the example is about even with a description. It looks broken. In my opinion there should be either the `AXA & SIE` column or the `IN KONTAKT BLEIBEN` column.

Comment (danyball) [reply src/components/30-organisms/footer/demo.js]: Ah sorry. Misunderstand that. I want to test that no emtpy accordion is rendered even for column 0. So both columns needs to be emtpy. But i could add the "in kontakt bleiben" column if that is better?

**Review by 01231 — APPROVED** [review]
Nice. Looks good!

**Pull request merged**